.github/workflows/macos.yml

```diff
@@ -213,3 +213,12 @@ jobs:
           set -o errexit -o pipefail -o noclobber -o nounset
           chmod +x test-*-package
           for TP in test-*-package; do "./$TP"; done
+
+  integration-test:
```

Comment:
Not sure I understand it

Reply:
Idea to do test that action is works without prior `checkout` step, once tests passed I will change branch to `main`

Reply:
Thest passed https://github.com/tamatebako/tebako/actions/runs/11806298189/job/32893116256

But will be broken next run because failed in main, after merge this test will be gree again